pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -919,6 +919,47 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
     )
     private int brokerMaxConnections = 0;
 
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "It relates to configuration \"WriteBufferHighWaterMark\" of Netty Channel Config. If the number of bytes"
+            + " queued in the write buffer exceeds this value, channel writable state will start to return \"false\"."
+    )
+    private int pulsarChannelWriteBufferHighWaterMark = 64 * 1024;
+
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "It relates to configuration \"WriteBufferLowWaterMark\" of Netty Channel Config. If the number of bytes"
+                + " queued in the write buffer is smaller than this value, channel writable state will start to return"
+                + " \"true\"."
+    )
+    private int pulsarChannelWriteBufferLowWaterMark = 32 * 1024;
+
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "If enabled, the channel will pause to deal with new request once the writer buffer is full, until it"
+                + " is changed to writable."
+    )
+    private boolean pulsarChannelPauseReceivingRequestsIfUnwritable = false;
+
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "After the connection is recovered from an unreadable state, the channel will be rate-limited for a"
+                + " period of time to avoid overwhelming due to the backlog of requests. This parameter defines how"
+                + " many requests should be allowed in the rate limiting period."
```

Comment:
"in the rate limiting period" doesn't seem to be accurate here since it's fixed to 1 second in the implementation. It seems that it should be mentioned that the unit is requests per second.